yellowbrick/classifier/top_features.py

```diff
@@ -0,0 +1,253 @@
+# classifier.top_features
+# Visualize top N features of a classifier
+#
+# Author:   Elaine Ayo <@ayota>
+# Created:  Tue May 23 14:10:42 2017 -0400
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: classifier.top_features.base.py [] $
+
+
+"""
+Visualize the top N features of classifier. This includes the N highest
+and the N lowest features.
+
+TODO:
+-   Docstrings
+-   Make quick method for producing plots for all classes using multiclass visualizer
+-   Add support for estimators with _feature_importance attributes
+-   Implement one-sided version that takes absolute value of coefs/features
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+import warnings
+import matplotlib.pyplot as plt
+
+from .base import ClassificationScoreVisualizer
+from ..exceptions import YellowbrickValueError
+from ..style import color_palette
+
+##########################################################################
+## Top Features Visualizer
+##########################################################################
+
+class TopFeaturesVisualizer(ClassificationScoreVisualizer):
```

Comment:
I suggest a rename to `TopNFeatures`; this might make it a bit easier to type (you can also alias the class to `TopFeaturesVisualizer` if you'd like).